include/open62541/server_pubsub.h

```diff
@@ -257,6 +257,7 @@ typedef struct {
     UA_Variant address;
     UA_KeyValueMap connectionProperties;
     UA_Variant connectionTransportSettings;
+    UA_Boolean enabled;
```

Comment:
Add some documentation what enabled means and when it takes effect.

Will this be written into when the component is manually enabled?